api-tests/src/steps/ipv-steps.ts

```diff
@@ -235,13 +266,27 @@ When(
   },
 );
 
-Then("I get a {string} identity", function (this: World, vot: string): void {
-  if (!this.identity) {
-    throw new Error("No identity found.");
-  }
+Then(
+  /I get a '(\w+)' identity(?: with(out)? a '([\w-]+)' VC)?/,
+  function (
+    this: World,
+    vot: string,
+    checkForAbsence: "out" | undefined,
+    expectedVc: string,
+  ): void {
+    if (!this.identity) {
+      throw new Error("No identity found.");
+    }
+    assert.equal(this.identity.vot, vot);
 
-  assert.equal(this.identity.vot, vot);
-});
+    if (expectedVc) {
+      this.vcs = decodeCredentialJwts(
```

Comment:
I've added it to the `/^I use the OAuth response to get my (identity|MFA reset result)$/` step since this is where we get the identity and to keep `there is a {VC} credential` more extendable